### ISSUE TITLE
Time tracking

### ISSUE THREAD

**Issue by KKSzymanowski** [MatteoGabriele/vue-analytics#16]
First off, I want to thank you for this package.

I want to propose adding time tracking to the available functionality.
It would use the API described here:
https://developers.google.com/analytics/devguides/collection/analyticsjs/user-timings

I'm no expert in JS but here's the proposed implementation:
```js
// src/track/time.js

/**
 * Time tracking
 * @param  {String} category
 * @param  {String} variable
 * @param  {Number} value
 * @param  {String} [label='']
 */
export default function (category, variable, value, label = '') {
  if (typeof window.ga === 'undefined') {
    return
  }

  window.ga('send', 'timing', category, variable, value, label)
}
```

And in the `src/index.js` you'd need to import the file:
```js
import trackTime from './track/time'
```
and register it
```js
const features = { trackEvent, trackPage, trackTime, set }
```

I can make a PR if you agree with my proposal.
I briefly tested it registering the function as `Vue.$ga.trackTime` and it appears to work.

**Comment by MatteoGabriele** [comment]
@KKSzymanowski yes please, feel free to make a PR.
Looks interesting!

thanks for this feature

**Comment by MatteoGabriele** [comment]
@KKSzymanowski  feature published in v2.4.0
